qcodes/instrument_drivers/tektronix/AWG70000A.py

```diff
@@ -122,10 +133,7 @@ def __init__(self,  parent: Instrument, name: str, channel: int) -> None:
                            label='Channel {} {} signal path'.format(channel, fg),
                            set_cmd='FGEN:CHANnel{}:PATH {{}}'.format(channel),
                            get_cmd='FGEN:CHANnel{}:PATH?'.format(channel),
-                           val_mapping={'direct': 'DIR',
-                                        'DCamplified': 'DCAM',
-                                        'AC': 'AC'}
-                           )
+                           val_mapping=_fg_path_val_map[self._parent.model])
```

Comment:
self._parent can now be replaced by the root instrument property but that it not essential

Reply:
Oh yes, that's right! Let's start using it.